templates/email/sutton/other-reported-bulky.html

```diff
@@ -0,0 +1,78 @@
+[%
+
+USE pounds = format('%.2f');
+PROCESS '_bulky_data.html';
+
+email_columns = 2;
+
+PROCESS '_email_settings.html';
+INCLUDE '_email_top.html';
+
+%]
+
+<th style="[% td_style %][% primary_column_style %]" id="primary_column">
+  [% start_padded_box | safe %]
+
+  <p style="[% p_style %]">Dear [% report.name %],</p>
+
+  <p style="[% p_style %]">[% email_summary %]</p>
+
+    [% INCLUDE '_council_reference.html' problem=report %]
+
+  <p style="[% p_style %]">
+    Date booking made: [% report_date %]
+  </p>
+
+  <p style="[% p_style %]">
+    Collection date: [% collection_date %]
+  </p>
+
+  <p style="[% p_style %]">
+    Items to be collected:
+    <ul>
+      [% FOR item IN item_list %]
+        <li>
+          [% item.value %]
+          [% IF item.message %]
+            <ul>
+              <li>
+                [% item.message %]
+              </li>
+            </ul>
+          [% END %]
+        </li>
+      [% END %]
+    </ul>
+  </p>
+
+[% IF payment %]
+  <p style="[% p_style %]">
+    Total cost: £[% pounds(payment / 100) %]
+  </p>
+[% END %]
+
+  <p style="[% p_style %]">
+    If you wish to cancel your booking, please visit <a href="[% cobrand.base_url %]/waste/[% report.get_extra_field_value('property_id') | uri %]/bulky_cancel">this link</a>.
+  </p>
+
+  <p style="[% p_style %]">
+    Items must be accessible and clearly visible from the front of your house, placed as close to the pavement as possible (but not on it). Our collections crews will not enter your home or retrieve items from your back garden.
+    If you live in a block of flats, you can place your items (on the morning of collection) in your communal bin store area. If the bin store has an access code you should have supplied that during the booking as we won't be able to collect otherwise.
+    Items must be out for collection by 6am on the collection day.
```

Comment:
Can we get the time here from the cobrand function `bulky_nice_collection_time`? Then kingston/sutton could share the email templates. In fact, the changes with Peterborough seem small enough, could we have them all in the one template with a few IFs?